engine.py

```diff
@@ -1282,8 +1298,7 @@ def _win32_get_proxy_window(self):
                     proxy_win_hwnd_found = (
                         self.__tk_aftereffects.win_32_api.find_windows(
                             stop_if_found=True,
-                            class_name="Qt5QWindowIcon",
-                            process_id=os.getpid(),
+                            window_text=win32_proxy_win_uuid,
```

Comment:
Since the window_text is set to a unique identifier, this seemed more stable and future-proof to me. Previously the class_name used during the lookup was a generic name provided by Qt5. That class_name value will probably change in the future if ShotGrid moves to Qt6.